submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/SubmissionResource.kt

```diff
@@ -33,6 +34,12 @@ class SubmissionResource(
         @PathVariable accNo: String
     ): Unit = submissionsWebHandler.deleteSubmission(accNo, user)
 
+    @DeleteMapping
+    fun deleteSubmissions(
```

Comment:
Please add the corresponding documentation for the new endpoint, i.e.:

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/docs/api/BioStudiesPublicAPI.yaml#L61